### ISSUE TITLE
No easy way to catch system calls

### ISSUE THREAD

**Issue by 10110111** [eteran/edb-debugger#250]
GDB has a command `catch syscall`. It can stop on each entry/exit/both of a syscall, be it a specific syscall or all syscalls.
Would be good if EDB also had this functionality.

**Comment by eteran** [comment]
Right. We should add an "execute until system call" feature. Which can offer a feature to filter to include/exclude some.

**Comment by 10110111** [comment]
I'd rather view it not as one-time "exec until" feature, but something like "set breakpoint with special condition", as it is in GDB — catch this condition.

**Comment by eteran** [comment]
Perhaps we do both? ;-)

**Comment by 10110111** [comment]
Won't harm indeed :)
